concepticondata/references/references.bib

```diff
@@ -3252,3 +3252,11 @@ @article{Rzymski2020
   year={2020},
   publisher={Nature Publishing Group}
 }
+
+@Book{Lenhard2015,
+  title     = {PPVT 4 (Deutsche Fassung)},
```

Comment:
Maybe we should spell out the abbreviation (Peabody Picture Vocabulary Test) because not everybody knows it.

Reply:
Good idea!